Add explanation on "Embed content" step (#1160)
ezsystems/developer-documentation: docs/tutorials/platform_beginner/7_embed_content.md

# Step 7 — Embed content

> Creating lists and detailed views of Content Types and their respective items often involves also loading their related resources.

Reviewer: ```suggestion
Creating lists and detailed views of Content Types and their respective items often involves loading related resources.
```

> In the following steps Rides will receive additional, related object: a Landmark, which will be displayed on Ride pages.

Reviewer: ```suggestion
In this step, you add a related object, a Landmark, which will be displayed on Ride pages.
```

### Create the RideController

> Since we will need to add additional information when Ride object is displayed, we will need to create a custom controller.

Reviewer: ```suggestion
You must provide additional information when the Ride object is displayed. 
This requires creating a custom controller.
```

> In this controller we will use `ContentService` to load related resources (Landmarks) for a particular Ride, and place it under `landmarksList` variable in template.

Reviewer: ```suggestion
The controller uses `ContentService` to load related resources (Landmarks) for a particular Ride.
```

I would move the mention of placing the controller in the template to the next section, something along the lines of:
"Now modify the Ride full view template to include a list of Landmarks, and the controller that you created above."

Author: I disagree. User-developer should in my opinion be informed what is the purpose of operations he/she is performing at the time they are introduced. 

Moving it further down the tutorial would mean that anyone doing it would not understand the steps, reducing it to a copy paste.

Reviewer: Paweł, I find your input extremely valuable and justified. I am not suggesting to eliminate the information, the change is merely about removing "we" from the sentence to bring the sentence closer to the general style of the docs, and to reorganize the information so that similar things stick together. 
Note that it is a question of moving this within the same article, the very next action.

Author: @dabrt Please don't get me wrong, I just have a different opinion ;) I respect you as doc maintainer and in the end, your word is law ;)

From my point of view, purpose of the action should be explained before it is done. The same goes for what our service class concepts are. It's easier to follow along if you can understand what it is for and why. That's why I would prefer the description to be before the implementation is shown.
